### PR TITLE
API: Add Path.addCircle, addRoundRect, close

### PR DESCRIPTION
**Pull request by lessp**
This adds bindings for `Path.` `addCircle`, `addRoundRect`, `close`. This is new ground for me, so please make sure I've not missed something! 😄 

https://api.skia.org/classSkPath.html#a2daa24dc5e277a487c80e2de4469bfb3

### CODE REVIEW

File: src/wrapped/bindings/SkiaWrappedBindings.re [hunk 1]
```diff
@@ -478,7 +478,21 @@ module M = (F: FOREIGN) => {
 
     let allocate = foreign("sk_path_new", void @-> returning(t));
     let delete = foreign("sk_path_delete", t @-> returning(void));
+    float;
```
Comment (Et7f3) [inline src/wrapped/bindings/SkiaWrappedBindings.re]: ?

**Review by Et7f3 — COMMENTED** [review]
Why a image is added but unused ?

File: src/wrapped/bindings/SkiaWrappedBindings.re (hunk 1 above)
Comment (lessp) [inline src/wrapped/bindings/SkiaWrappedBindings.re]:
```suggestion
```

Comment (lessp) [reply src/wrapped/bindings/SkiaWrappedBindings.re]: Must have gotten there by accident! Thanks!

**Comment by lessp** [comment]
> Why a image is added but unused ?

Removed it and added it to `.gitignore`, thanks!

**Comment by lessp** [comment]
> > Why a image is added but unused ?
> 
> Removed it and added it to `.gitignore`, thanks!

Anything else holding this back @Et7f3?

File: src/Skia.re [hunk 2]
```diff
@@ -459,6 +459,9 @@ module RRect = {
 };
 
 module Path = {
+  type pathDirection = SkiaWrapped.Path.pathDirection;
+  let pathDirection = SkiaWrapped.Path.pathDirection;
+
```
Comment (Et7f3) [inline src/Skia.re]:
```suggestion
```

File "src/Skia.re", line 462, characters 2-53:
Error (warning 34): unused type pathDirection.
File "src/Skia.re", line 463, characters 6-19:
Error (warning 32): unused value pathDirection.

File: example/TestApp.re [hunk 3]
```diff
@@ -34,6 +34,13 @@ let draw = canvas => {
   Path.lineTo(path, 590., 50.);
   Path.cubicTo(path, -490., 50., 1130., 430., 50., 430.);
   Path.lineTo(path, 590., 430.);
+  Path.addCircle(path, 100., 300., 100., Clockwise);
+
+  let roundRect = Rect.makeLtrb(300., 400., 240., 280.);
+  Path.addRoundRect(path, roundRect, 25., 25., Clockwise);
+  Path.addCircle(path, 100., 300., 100., Clockwise);
+
+  Path.close(path);
```
Comment (Et7f3) [inline example/TestApp.re]:
```suggestion
  Path.close(path);
  Path.addCircle(path, 100., 300., 100., Clockwise);

  let roundRect = Rect.makeLtrb(300., 400., 240., 280.);
  Path.addRoundRect(path, roundRect, 25., 25., Clockwise);
  Path.addCircle(path, 100., 300., 100., Clockwise);
```

For me I don't see the close take effect expect I move before other call. Is it a bug only on my computer ? And you have two call with the same coordinates is it intended ? It make the stoke a bit more thick.

**Comment by Et7f3** [comment]
Overwise seem good to merge

File: example/TestApp.re (hunk 3 above)
Comment (Et7f3) [reply example/TestApp.re]: I think it is ok:
`close() has no effect if SkPath is empty or last SkPath SkPath::Verb is kClose_Verb.` addCircle should add a closed shape.

**Comment by Et7f3** [comment]
I see in the man that their is a default value to ClockWise. Maybe when you re export you could define a function with a default arg. :) Otherwise seem good to merge.

File: example/TestApp.re [hunk 4]
```diff
@@ -34,6 +34,12 @@ let draw = canvas => {
   Path.lineTo(path, 590., 50.);
   Path.cubicTo(path, -490., 50., 1130., 430., 50., 430.);
   Path.lineTo(path, 590., 430.);
+  Path.close(path);
+  Path.addCircle(path, 100., 300., 100., ());
```
Comment (Et7f3) [inline example/TestApp.re]:
```suggestion
  Path.addCircle(path, 100., 200., 75., ());
```
So we can see both circle :)

**Review by Et7f3 — APPROVED** [review]

**Comment by lessp** [comment]
Perfect, thanks @Et7f3!

**Pull request merged**